BUILDING_ON_WINDOWS.md

```diff
@@ -82,19 +93,33 @@ Note: This installation will take about 200 MB of disk space.
 
 Install [conan 2](https://conan.io/downloads.html) and make sure it's in your `PATH` (default setting).
 
+<details>
+   <summary>How to add conan 2 to PATH</summary>
+   
+1. Type "path" in the start menu and click on the "Edit the system environment variables" result that shows up (this should open up the 'System Properties' window with the 'Advanced' tab selected).
+2. Press on the "Environment Variables..." button near the bottom right of the 'System Properties' window to open up the 'Environment Variables' window.
+3. Within 'Environment Variables', look at the bottom half of the window where it lists 'System variables' and double click on the variable named 'Path' to open up the 'Edit environment variable' window.
+4. Press the 'New' button near the top right to create a new environment variable.
+5. Open up your terminal with the Visual Studio environment variables (e.g. `x64 Native Tools Command Prompt for VS 2022`) and type "where conan" to find the file path (the folder that contains the conan.exe) to add.
+6. Add conan 2's file path (e.g. `C:\Users\willg\AppData\Roaming\Python\Python311\Scripts`) to the blank text box that shows up.
+7. Press the "OK" button.
+
+</details>
+
 Then in a terminal, configure conan to use `NMake Makefiles` as its generator:
 
 1. Generate a new profile  
    `conan profile detect`
 
 #### Build
 
-Open up your terminal with the Visual Studio environment variables (e.g. `x64 Native Tools Command Prompt for VS 2022`), cd to the cloned c2 directory and run the following commands:
+Open up your terminal with the Visual Studio environment variables (e.g. `x64 Native Tools Command Prompt for VS 2022`), cd to the cloned chatterino2 directory and run the following commands:
 
 1. `mkdir build`
 1. `cd build`
 1. `conan install .. -s build_type=Release -c tools.cmake.cmaketoolchain:generator="NMake Makefiles" --build=missing --output-folder=.`
 1. `cmake -G"NMake Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE="conan_toolchain.cmake" -DCMAKE_PREFIX_PATH="C:\Qt\5.15.2\msvc2019_64" ..`
+1. `cd ..`
```

Comment:
Based on my recollection, I couldn't use 'nmake' in the build folder.

Below is what I wrote for Felanbird's comment on the same 'cd ..':

"If I recall correctly, it wouldn't let me use 'nmake' in the build folder. I had to be in the chatterino2 folder for it to work.

Right now, when I try to do 'nmake' in the build folder, I get
'NMAKE : fatal error U1064: MAKEFILE not found and no target specified
Stop.'

However, I'm not managing to build successfully at the moment (I keep getting LINK and NMAKE fatal errors that I have not currently figured out) even with 'nmake' in the chatterino2 folder because I get 'fatal error' for LINK and NMAKE.

My best guess is because I messed up something with either the GitHub branches, git commands, and/or something in VS Code (which I use over Visual Studio Community right now) when I tried to get the copy of the original repository in my fork without being 'commits ahead'."

Is that just a me issue?

Reply:
Oh, I overlooked that (or I think it didn't show on mobile), let's discuss this there then.

Reply:
I removed the 'cd ..'